.asf.yaml

```diff
@@ -15,11 +15,15 @@
 # specific language governing permissions and limitations
 # under the License.
 
+# All configurations could be found here: https://github.com/apache/infrastructure-asfyaml/
+
 github:
   description: A software library of stochastic streaming algorithms, a.k.a. sketches.
   homepage: https://datasketches.apache.org
-  ghp_branch: gh-pages
-  ghp_path: /docs
```

Comment:
.. until we really need it. I guess most of our docs would either go to https://datasketches.apache.org/ or https://docs.rs/datasketches.